docs/versions.json

```diff
@@ -1,8 +1,3 @@
 [
-  "2.6.0",
-  "2.4.0",
-  "2.3.0",
-  "2.1.1",
-  "2.0.0",
-  "1.10.3"
-]
+  "1.x"
+]
```

Comment:
Here and in the `_category_.json` files the new lines at the end of the file are missing.